### DISCUSSION TITLE
QuasarV1 Webpack support

### DISCUSSION THREAD

**Discussion by gugugulard** [quasarframework/quasar#14544] [General - Components / Directives / etc] [answered]
Hi!

I'd like to confirm if QuasarV1 works with webpack 5 or just 4? I checked https://github.com/quasarframework/quasar/issues/8102 and it seems the webpack 5 upgrade is specific to QuasarV2 only.

Many thanks!

**Comment by MilosPaunovic** [comment, marked as answer]
Q1 requires Webpack 4.

**Reply by gugugulard** [reply]
noted on this and many thanks!
